### ISSUE TITLE
Grasshopper PublishAnimation Animation behaviour 

### ISSUE THREAD

**Issue by gesievi** [Archtica/Radii#38]
When selecting play and reverse it just plays the animation forward. 
Reverse is only executed when coupled with repeat in this case the animation repeats the animation in both directions cyclical.

**Comment by Archtica** [comment]
Yes, that it how it is at the moment. If you have return only it will return to the starting point after reaching goal. With return and reverse it will reverse to it's starting point. I know it's a bit counter intuitive. Maybe I should only allow to select reverse when return is selected otherwise it's not an option.

**Comment by gesievi** [comment]
The way i can observe it functioning 
Play = Plays animation once
Replay=Play + replay --> runs the animation in loop from the starting point
Reverse = Play + reverse --> runs the animation once
                Play + reverse + return --> runs the animation once returns to start
                 Play + reverse + replay --> runs the animation forward and backward in                        
                 loop
                 
Return = Play + reverse = plays once and 

What i found counter intuitive is that play+reverse is not running the animation in reverse, this seems like a bug to me.

**Comment by Archtica** [comment]
> The way i can observe it functioning Play = Plays animation once Replay=Play + replay --> runs the animation in loop from the starting point Reverse = Play + reverse --> runs the animation once Play + reverse + return --> runs the animation once returns to start Play + reverse + replay --> runs the animation forward and backward in loop
> 
> Return = Play + reverse = plays once and
> 
> What i found counter intuitive is that play+reverse is not running the animation in reverse, this seems like a bug to me.

Yes, I agree. I will make it so that if reverse is selected then return is implied or only allow reverse to be selected if return is selected first. One or the other.
